FeedSX/store/actions/toast.ts

```diff
@@ -1,15 +1,16 @@
 import {Dispatch} from '@reduxjs/toolkit';
 import {Alert} from 'react-native';
-import { SHOW_TOAST } from '../types/loader';
+import {SHOW_TOAST} from '../types/loader';
 
 // show toast message action
-export const showToastMessage = (payload?: any) => async (dispatch: Dispatch) => {
-  try {
-    return await dispatch({
-      type: SHOW_TOAST,
-      body: payload
-    });
-  } catch (error) {
-    Alert.alert(`${error}`);
-  }
-};
+export const showToastMessage =
+  (payload?: any) => async (dispatch: Dispatch) => {
```

Comment:
why `any`?

Reply:
this is related to dispatch as any, have to define according to that.